uniro_frontend/src/pages/navigationResult.tsx

```diff
@@ -118,7 +126,9 @@ const NavigationResultPage = () => {
 				risks={risks.data}
 				topPadding={topBarHeight}
 				bottomPadding={sheetHeight}
+				handleCautionMarkerClick={handleCautionMarkerClick}
 				currentRouteIdx={currentRouteIdx}
+				setDangerRouteIdx={setCautionRouteIdx}
```

Comment:
Props의 이름이 부적절한 것 같습니다!
```suggestion
setCautionRouteIdx={setCautionRouteIdx}
```